modules/siddhi-extensions/map/src/test/java/org/wso2/siddhi/extension/map/CreateFromJSONFunctionExtensionTestCase.java

```diff
@@ -120,12 +119,88 @@ public void receive(Event[] inEvents) {
 
         InputHandler inputHandler = executionPlanRuntime.getInputHandler("inputStream");
         executionPlanRuntime.start();
-        inputHandler.send(new Object[]{"IBM", 100, 100l});
-        inputHandler.send(new Object[]{"WSO2", 200, 200l});
-        inputHandler.send(new Object[]{"XYZ", 300, 200l});
-        SiddhiTestHelper.waitForEvents(100, 3, count, 60000);
-        Assert.assertEquals(3, count.get());
+        inputHandler.send(new Object[]{"IBM",100,100l});
+        inputHandler.send(new Object[]{"WSO2",200,200l});
+        inputHandler.send(new Object[]{"XYZ",300,200l});
+        SiddhiTestHelper.waitForEvents(100,3,count,60000);
+        Assert.assertEquals(3,count.get());
         Assert.assertTrue(eventArrived);
         executionPlanRuntime.shutdown();
```

Comment:
Please fix all formating chages you have done upto this point. Please review your PR before sending it.